docs/index.html

```diff
@@ -792,6 +794,55 @@ <h3 class="i18n" id="trx_color">Custom Colors</h3>
     </p>
     <hr>
 
+    <h2>Searching for Information</h2>
+        <div class="row">
+        </div>
+    
+        <h3>Finding Transactions</h3>
+            <div class="row">
+            </div>
+            <p>
+                The MMEX desktop application provides several utilities to help users find specific transactions.  The "Notes" field of transactions can be searched from the account view. At the bottom of the transaction listing in
+                the account view, input your search query in the text box to the right of the magnifying glass. Click the magnifying
+                glass or press the Enter key to search the transactions in that account. The search is case sensitive.
+            </p>
+            
+            <p>
+                The most powerful tool for finding particular transaction information is the Transaction Reports feature, which is 
+                described in more detail <a class="i18n" href="#section21.1">here</a>.
```

Comment:
Bad idea to use dynamically created href.
I've changed it 
https://github.com/vomikan/moneymanagerex/blob/master/docs/en_US/index.html#L718

Reply:
I had initially tried to use a relative (non-dynamic) href pointing to that section, but when I reload index.html the link is broken. When I make the change you referenced, I cannot navigate to the other section of the document using the hyperlink.

 In light ofid attributes within the headers being deprecated (https://github.com/moneymanagerex/moneymanagerex/pull/1936#discussion_r248239950), is there another strategy being used to href sections?

Reply:
@vomikan any suggestions on how to change the hrefs?

Reply:
Add to the target section / paragraph an unique id (like `<p id="my_target">`) then use it here (as `... href="#my_target"...`. Use some descriptive name of course.

sectionXX.X ids are autogenerated and XX.XX can change when new section is added or removed from document.